src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -169,6 +170,24 @@ public int RegionSource
             set => this[RegionSourceKey] = (value).ToString(CultureInfo.InvariantCulture);
         }
 
+        public bool? IsValidUserProvidedRegion
+        {
+#pragma warning disable CA1305 // .net standard does not have an overload for ToString() with Culture
```

Comment:
>.net standard does not have an overload for ToString() with Culture [](start = 34, length = 67)

aside: [https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings?redirectedfrom=MSDN#stringtoupper-and-stringtolower](https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings?redirectedfrom=MSDN#stringtoupper-and-stringtolower)

Reply:
If I use toUpper, it messes up with code [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c98af5f29b76077a9bf596b3e0c0dc5d78244755/src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs#L132) when it converts the bool to bitwise. So leaving it as is.